### ISSUE TITLE
A picture is worth a thousand words

### ISSUE THREAD

**Issue by shujaatak** [wangwenx190/framelesshelper#8]
Some screenshots in the readme file would greatly benefit this repository.

**Comment by wangwenx190** [comment]
Yes, you are right. But sadly I don't have appropriate demo projects. Can you make some screenshots and submit a pull request?

By the way, is the resize area issue solved?

**Comment by shujaatak** [comment]
Sure, it would be such an honor for me to submit a pull request once I make some catchy demos.

For now, the screenshot of the existing empty frame-less window would be sufficient to feed the hungry eyes of the visual learners.

Yes, the resize area issue is solved.

I think, still there is a small issue( if it might be called an issue), that when you resize the window upward using the top border you can see the white flashes near the top border region ( I have put around 5 dummy buttons with a gridlayout in the form).

**Comment by wangwenx190** [comment]
> 
> For now, the screenshot of the existing empty frame-less window would be sufficient to feed the hungry eyes of the visual learners.
> 

It's only a blank window right now. Would it be helpful?

> 
> when you resize the window upward using the top border you can see the white flashes near the top border region
>

I have a bug report long ago which said the white background of the frameless window can be seen by the user when resizing rapidly. I feel the problem you reported is the same with this one. As what I said, I got this report long ago, maybe a month ago, I just don't know what caused this. And I found that if you don't apply any QSS to the frameless window, the issue can be solved. Don't know why.
